src/commands/dummy.ts

```diff
@@ -0,0 +1,17 @@
+// import * as express from "express";
+// import * as Promise from "bluebird";
+
+// function usage() {
+//   return "dummy [args...]";
+// }
+
+// export function execute(
+//   _app: express.Application,
+//   args: string[]
+// ): Promise<void> {
+//   // grab your args as an array, like so...
+//   const [arg1, arg2, argN] = args;
+
+//   // return a promise which resolves to void...
+//   return Promise.resolve();
+// }
```

Comment:
Can you please delete this file? If you still need to show an example command, please move it into the README.

Reply:
I have kept it - but made it actually executable, as a working example